.github/workflows/test.yml

```
@@ -9,15 +9,15 @@ jobs:
     steps:
     - name: Checkout code
       uses: actions/checkout@v3
-    - uses: actions/setup-node@v3
+    - name: Setup Node
+      uses: actions/setup-node@v3
       with:
         node-version: '18'
```

Let's speed up CI by using the npm cache
```suggestion
        node-version: '18'
        cache: 'npm'
```

---

That appears to require `package-lock.json`.

---

I don't think it's a problem, because this CI doesn't run that often, and in fact we want the CI to run against the newest version of the dependencies available at the time it runs that match the semvers declared in package.json.